src/storybook/stand-alone-documentaion/catalog/assets/index.js

```diff
@@ -0,0 +1,3 @@
+import emptyStateExample from "./emptyStateExample.svg";
+
+export { emptyStateExample };
```

Comment:
Is it possible to use svg as an image in MDX?

Reply:
Refactored this part